src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+
+        {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+
+        <p className="Person__partner">
+          {isMarried === true && sex === 'f' && `${partnerName} is my husband`}
+          {isMarried === true && sex === 'm' && `${partnerName} is my wife`}
+          {isMarried === false && 'I am not married'}
```

Comment:
I understand that you can change 'isMarried === true' to just 'isMarried' and 'isMarried === false' to '!isMarried'.